src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -415,14 +421,47 @@ private function isDatabasePresent()
 
     /**
      * Check if the schema is already created.
-     * If we found at least one table, it means the schema exists.
-     *
-     * @return bool
+     * We use the Doctrine Migrations table for the check.
      */
-    private function isSchemaPresent()
+    private function isSchemaPresent(): bool
+    {
+        $schemaManager = $this->entityManager->getConnection()->createSchemaManager();
+
+        return $schemaManager->tablesExist([$this->tableMetadataStorageConfiguration->getTableName()]);
+    }
+
+    private function dropWallabagSchemaOnly(): void
     {
+        $this->runCommand('doctrine:schema:drop', ['--force' => true]);
+
+        $connection = $this->entityManager->getConnection();
+        $databasePlatform = $connection->getDatabasePlatform();
         $schemaManager = $this->entityManager->getConnection()->createSchemaManager();
 
-        return \count($schemaManager->listTableNames()) > 0 ? true : false;
+        try {
+            $schemaManager->dropTable($databasePlatform->quoteIdentifier($this->tableMetadataStorageConfiguration->getTableName()));
+        } catch (TableNotFoundException $exception) {
+        }
+
+        // @TODO: remove below code when the schema is fixed
+        foreach ($this->entityManager->getMetadataFactory()->getAllMetadata() as $metadata) {
+            try {
+                $schemaManager->dropTable($databasePlatform->quoteIdentifier($metadata->getTableName()));
+            } catch (TableNotFoundException $exception) {
+            }
+
+            foreach ($metadata->getAssociationMappings() as $associationMapping) {
+                if (ClassMetadataInfo::MANY_TO_MANY !== $associationMapping['type']) {
+                    continue;
+                }
+
+                try {
+                    if (isset($associationMapping['joinTable']['name'])) {
+                        $schemaManager->dropTable($databasePlatform->quoteIdentifier($associationMapping['joinTable']['name']));
+                    }
+                } catch (TableNotFoundException $exception) {
+                }
+            }
+        }
```

Comment:
all this is needed because the Doctrine metadata are not in sync with the schema created by migrations...
run `bin/console doctrine:schema:validate` to see it broken.
I plan to sync metadata and schema but this will require a dedicated work that will need discussion as it may involve big migrations (run `bin/console doctrine:schema:validate -v` or `bin/console doctrine:migrations:diff` to see the inconsistencies)
for example using PostgreSQL the command `doctrine:schema:drop` doesn't remove 2 tables